### DISCUSSION TITLE
MacOS x86_64 build error

### DISCUSSION THREAD

**Discussion by CocoaSamuel** [audacity/audacity#5452] [Development]
### Bug description

xcode:14.2
macos:13.4
static_assert(is_base_of_v<is_integral<double>,
   Every<Fn<is_integral>, Example>>); 
error :Static_assert failed due to requirement 'is_base_of_v<std::is_integral<double>, TypeList::Every<TypeList::Fn<std::is_integral>, TypeList::List<int, double>>>'

### Steps to reproduce

1. open Audacity.xcodeproj
2. build error

### Expected behavior

build success

### Audacity Version

current master / alpha / nightly build

### Operating system

macOS

### Additional context

![image](https://github.com/audacity/audacity/assets/71318196/373ae631-82e1-43c1-b4d2-6512a8d69107)

**Comment by Paul-Licameli** [comment]
Are you using a different version of the standard library, which does not correctly implement `std::conjunction` ? 

https://en.cppreference.com/w/cpp/types/conjunction

**Comment by crsib** [comment]
There is a bug in libc++ implementation on certain XCode versions.

**Reply by mohd-akram** [reply]
Is there a workaround for this?

**Comment by clausecker** [comment]
Same problem on FreeBSD 13.2.
